### DISCUSSION TITLE
Does the dynamic route directory name for locale have to be [locale]?

### DISCUSSION THREAD

**Discussion by 58bits** [amannn/next-intl#1220] [Q&A]
Does the dynamic route directory for locale have to be [locale]? We have an existing application and would like to switch to next-intl, but our current top-level directory for the language segment is [lng] and not [locale]. 

Can we use [lng]?

**Comment by amannn** [comment]
For the time being, the `[locale]` segment is required—yes.

I'd like to make this more dynamic, but the ideal solution depends on https://github.com/amannn/next-intl/issues/663.

**Reply by 58bits** [reply]
Okay understood - and thanks for the reply @amannn
